docs/README.md

```diff
@@ -39,6 +39,20 @@ yarn run export-server
 
 You can find the source of the documentation inside the `pages/versions` directory. Documentation is mostly written in markdown with the help of some React components (for Snack embeds, etc). The routes and navbar are automatically inferred from the directory structure within `versions`.
 
+### Editing Code
+
+Docs are made in Next.js and TypeScript. If you need to make code changes, follow steps from [Running locally](#running-locally) section, then open a separate terminal and run TypeScript compiler in watch mode - it will watch your code changes and notify you about errors.
```

Comment:
```suggestion
The docs are written with Next.js and TypeScript. If you need to make code changes, follow steps from [Running locally](#running-locally) section, then open a separate terminal and run TypeScript compiler in watch mode - it will watch your code changes and notify you about errors.
```